shared/settings/account/add-modals.tsx

```diff
@@ -88,8 +89,12 @@ export const Email = () => {
           searchable={searchable}
           onChangeSearchable={onChangeSearchable}
           onContinue={onContinue}
-          icon={
-            <Kb.Icon type={Styles.isMobile ? 'icon-email-add-96' : 'icon-email-add-64'} style={styles.icon} />
+          iconType={
+            Styles.isMobile
```

Comment:
`Styles.isMobile && Platform.isLargeScreen ? "big" : "small"`

Reply:
Is that why the dimension styles were on them?

Reply:
Yes

Reply:
(I'm fine with removing them, rather do it in `Icon` than add more plumbing to make it work with your changes here)

Reply:
Works for me.